tests/integ/test_track_from_training_job.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
this should be done automatically, i created an issue

Reply:
agree